pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -630,7 +635,19 @@ spec:
       - {{ $v | quote }}
       {{- end }}
       {{- end }}
+    {{- else -}}
+    {{- $poolUids := keys .ListItems.cvolPoolList.pools | randomize }}
+    {{- $replicaCount := .Config.ReplicaCount.value | int64 -}}
+    repeatWith:
+      resources:
+      {{- range $k, $v := $poolUids }}
+      {{- if lt $k $replicaCount }}
+      - {{ $v | quote }}
+      {{- end -}}
+      {{- end -}}
+    {{- end -}}
   task: |
+    {{- $replicaAffinityName:= .TaskResult.creategetpvc.replicaAntiAffinity -}}
```

Comment:
here as well, name `replicaAffinityName` is not correct

Reply:
done